sources/packages/api/src-sql/Types/alter-program-info-status.sql

```diff
@@ -0,0 +1,19 @@
+-- Adjust the case to be user-friendly starting with proper case as decided by the team.
+ALTER TYPE sims.program_info_status RENAME VALUE 'required' TO 'Required';
+
+ALTER TYPE sims.program_info_status RENAME VALUE 'not required' TO 'Not Required';
+
+ALTER TYPE sims.program_info_status RENAME VALUE 'completed' TO 'Completed';
+
+-- Add new enum options.
+ALTER TYPE sims.program_info_status
+ADD
+  VALUE 'Submitted'
+AFTER

```

Comment:
I am not sure if I follow completely 😄 
My intention was to organize the enum in the same way that it would be organized if it would be created at the same moment for the first time, as shown below. Right now, there is no concern about removing any enum value.
Please let me know what I am missing here.

![image](https://user-images.githubusercontent.com/61259237/128942330-212b4d3d-8b28-4502-adaf-0e2b28c7e863.png)

Reply:
I get it @andrewsignori-aot. I am saying if in the future, Not Required value is removed from the enum, then the alter statement will fail during the migration run. Did you get it now?

Reply:
It would not be true because the migration to do the mentioned change would be on top of this. That is the beauty of migrations, it always moves forward 😉.  The removal of an enum value would be a new migration on top of that.